src/main/java/org/opentripplanner/routing/api/request/preference/StreetPreferences.java

```diff
@@ -49,6 +50,7 @@ private StreetPreferences() {
     this.maxAccessEgressDuration = durationForStreetModeOf(ofMinutes(45));
     this.maxDirectDuration = durationForStreetModeOf(ofHours(4));
     this.routingTimeout = Duration.ofSeconds(5);
+    this.maxAccessEgressStopCount = 0;
```

Comment:
We now have 3 parameters for accessEgress (accessEgressPenalty, maxAccessEgressDuration, maxStopCount) so we should probably put them into its own Preference class and not repeat the accessEgress prefix. 

Would a better default value for `maxAccessEgressStopCount` be 500? I understand that you probably set it to 0 to not change the behaviour, but: "Making OTP easy to configure (good defaults) is preferred over unchanged behaviour"